homeassistant/components/config/entity_registry.py

```diff
@@ -96,7 +98,8 @@ def websocket_update_entity(hass, connection, msg):
     """
     registry = er.async_get(hass)
 
-    if msg["entity_id"] not in registry.entities:
+    entity_id = msg["entity_id"]
+    if entity_id not in registry.entities:
```

Comment:
Shouldn't you just store the entry here in a var so you don't need to fetch it inside the `disabled_by` if ?